WordPress/WordPressTest/NSManagedObject+Fixture.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+extension NSManagedObject {
+
+    /// Loads the contents of any given JSON file into a new NSManagedObject instance.
```

Comment:
Nitpick: Just so documentation formats it as code

```suggestion
    /// Loads the contents of any given JSON file into a new `NSManagedObject` instance.
```